pkg/database/testing.go

```diff
@@ -111,11 +112,26 @@ func createTestingHiveDB() *DB {
 }
 
 func testingMaxComputeConfig() *gomaxcompute.Config {
+	endpoint := test.GetEnv("SQLFLOW_TEST_DB_MAXCOMPUTE_ENDPOINT", "http://service-maxcompute.com/api")
+	urlAndArgs := strings.Split(endpoint, "?")
+	// accept format like http://service-maxcompute.com/api
+	// if got service-maxcompute.com/api?curr_project=xxxx&scheme=http, reformat to http://service-maxcompute.com/api
+	if len(urlAndArgs) == 2 {
```

Comment:
Should we assert here that `len(urlAndArgs)==2`? Otherwise, we need an `else` to handle the case that it is not 2.